repo-teams.txt

```diff
@@ -0,0 +1,26 @@
+vault-plugin-auth-alicloud               vault-eco
+vault-plugin-auth-azure                  vault-eco
+vault-plugin-auth-centrify               vault-eco
+vault-plugin-auth-cf                     vault-eco
+vault-plugin-auth-gcp                    vault-eco
+vault-plugin-auth-jwt                    vault-eco
+vault-plugin-auth-kerberos               vault-eco
+vault-plugin-auth-kubernetes             vault-eco
+vault-plugin-auth-oci                    vault-eco
+vault-plugin-database-couchbase          vault-eco
+vault-plugin-database-elasticsearch      vault-eco
+vault-plugin-database-mongodbatlas       vault-eco
+vault-plugin-database-oracle             vault-eco
+vault-plugin-database-redis              vault-eco
+vault-plugin-database-redis-elasticache  vault-eco
+vault-plugin-database-snowflake          vault-eco
+vault-plugin-secrets-ad                  vault-eco
+vault-plugin-secrets-alicloud            vault-eco
+vault-plugin-secrets-azure               vault-eco
+vault-plugin-secrets-gcp                 vault-eco
+vault-plugin-secrets-gcpkms              vault-eco
+vault-plugin-secrets-kubernetes          vault-eco
+vault-plugin-secrets-kv                  core
```

Comment:
This team may need to be `foundations`?

Reply:
oh, yeah I forgot to check on that. Thanks!

Reply:
I asked in the vault slack. I will merge this now and we can update when I get a response.